src/components/graph-view.js

```diff
@@ -904,10 +902,15 @@ class GraphView extends Component {
   renderView = () => {
     var nodes = this.props.nodes;
     var edges = this.props.edges;
-
     // Update the view w/ new zoom/pan
-    const view = d3.select(this.view)
-      .attr("transform", this.state.viewTransform);
+    const view = d3.select(this.view);
+    const viewNode = view.node();
+    if (viewNode) {
+      const { k, x, y } = this.state.viewTransform;
+      view.attr("transform", this.state.viewTransform);
+      // viewNode.setAttribute('transform', `translate(${x},${y}) scale(${k})`);
+      viewNode.style.transform = null;
```

Comment:
d3 (or something else), for whatever reason, is adding a transform style when the transform attribute is modified in any way. Whether we use pure JS to change the attribute, or if we use d3 to set the attribute something adds a transform style that overrides our transform attribute. Setting that style to null stops that from happening.